### ISSUE TITLE
"Publish in the future" will happen randomly, even when you don't select it

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#8251]
**Describe the bug**
I upload videos. Sometimes I wonder where they are. When this happens, I have learned that I must "publish" the video, despite having **never** selected to publish in the future - I _never_ want to do that!

**To Reproduce**
1. Upload a video
2. See if it displays
3. If it doesn't?
4. You have to go manually publish it

**Expected behavior**
It not do that.

**Solution**
1. Fix the problem with intermittently putting the video on a publish schedule, even though none was selected.
2. Give us a global way to disable this feature that very few people are using, like, we don't want our users to use it - EVER! (**EVER!!!!!!**)

**Comment by walt93** [comment]
UPDATE: I uploaded four videos, and have been playing a game of whack-a-mole with them, I uploaded none of them with publish-in-the-future, observed that two of them were scheduled for publish-in-the-future, and while publishing them and editing their metadata, I have watched two of them flip BACK into publish-in-the-future, I'm having a hard time characterizing it other than to say it seems fairly broken… :-(

**Comment by walt93** [comment]
It's hard to characterize because I wasn't paying super close attention. I believe it looked like this, it started with one in un-published state, but I had some edits to do (changing video owner, etc.) and while making the edits I observed that some had flipped to publish-in-the-future. I'm not 100% clear which step I took that caused them to flip.

These were all short (<3 minute) pieces so I'm pretty sure they were all finished encoding.
